app-i18n/uchardet/uchardet-9999.ebuild

```diff
@@ -7,10 +7,7 @@ inherit cmake-utils git-r3
 
 DESCRIPTION="An encoding detector library"
 HOMEPAGE="https://www.freedesktop.org/wiki/Software/uchardet/"
-EGIT_REPO_URI=(
-	https://anongit.freedesktop.org/git/${PN}/${PN}.git
-	git://anongit.freedesktop.org/${PN}/${PN}
-)
+EGIT_REPO_URI="https://anongit.freedesktop.org/git/${PN}/${PN}.git"
```

Comment:
@Coacher, it's not about 'reading' git traffic. It's about MITM when someone puts a fake git server with modified code that e.g. has backdoors. Even though we don't really support live ebuilds, we do not want to expose people to this kind of danger by default.